quickstarts/microsoft.web/function-premium-vnet-integration/main.bicep

```diff
@@ -0,0 +1,149 @@
+@description('Location for all resources except Application Insights.')
+param location string = resourceGroup().location
+
+@description('Location for Application Insights.')
+param appInsightsLocation string
+
+@description('The language worker runtime to load in the function app.')
+@allowed([
+  'node'
+  'dotnet'
+  'java'
+])
+param runtime string = 'node'
+
+@description('The name of the function app that you wish to create.')
+param appName string = 'fnapp${uniqueString(resourceGroup().id)}'
+
+@allowed([
+  'Standard_LRS'
+  'Standard_GRS'
+  'Standard_RAGRS'
+])
+@description('Storage Account type')
```

Comment:
move to the top

Reply:
decompiler now does this automatyically.